### DISCUSSION TITLE
client crash during plotting

### DISCUSSION THREAD

**Discussion by khudok** [Chia-Network/chia-blockchain#4445] [Q&A]
If one were to accidentally close the client during plotting, is there any clean up required? or will the plotting resume? Any pointers on how to clean up the abandoned plotting? 

thanks,

**Comment by SackofFluor** [comment]
You cant continue abandoned plots!

The only chance you have if the gui crashes is, to see whether there is still strong usage on the CPU, SSD and maybe the network. If that is still running you have good chances that the plotting wasn't cancelled and will finish (even without being mentioned in the GUI at all).
But truly abandoned plots are gone and you have to start over again

**Reply by khudok** [reply]
thank you. as for the clean up, I will search the FAQ for guidance on what temp files need to be removed.
